css/types/basic-shape.json

```diff
@@ -273,25 +273,19 @@
               "firefox": {
                 "version_added": "63",
                 "partial_implementation": true,
-                "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.clip-path-path.enabled",
-                    "value_to_set": "true"
-                  }
+                "notes": [
```

Comment:
So these are now all notes as suggested. Note that using this approach we lose the preference as flags.

Reply:
Oh, interesting! Writing the notes shows a much more complex story than I gleaned from the PR commentary. Here's how I'd represent this:

```json
[
  {
    "version_added": "63",
    "partial_implementation": true,
    "notes": [
      "From version 97, the <code>d</code> SVG presentation attribute supports <code>path()</code>.",
      "From version 71, the <code>clip-path</code> property supports <code>path()</code>.",
      "Before version 71, only the <code>offset-path</code> property supports <code>path()</code>."
    ]
  },
  {
    "version_added": "91",
    "version_removed": "97",
    "partial_implementation": true,
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.d-property.enabled",
        "value_to_set": "true"
      }
    ],
    "notes": [
      "The <code>d</code> SVG presentation attribute, <code>clip-path</code> CSS property, and <code>offset-path</code> CSS property support <code>path()</code>."
    ]
  },
  {
    "version_added": "64",
    "version_removed": "71",
    "partial_implementation": true,
    "flags": [
      {
        "type": "preference",
        "name": "layout.css.clip-path-path.enabled",
        "value_to_set": "true"
      }
    ],
    "notes": [
      "The <code>clip-path</code> and <code>offset-path</code> properties support <code>path()</code>."
    ]
  }
]
```

There's a lot going here! Some highlights:

- Unflagged support goes first, even though the flag stuff is more recent. The idea here is that the most expansive support goes first, not necessarily the most recent.
- Notes are sorted with most recent first, since the other notes should become increasingly less interesting as time goes on.
- The notes attached to the flagged statements don't need to specify version numbers, since it applies to the whole range covered by that statement.
- The notes have been reworded to lose the ambiguous "it" which had me completely lost before long.

All that said, I'm not sure if this information will copy over to the properties themselves very well. I'd be okay with dropping the older of the two flagged statements (in following [this guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#removal-of-irrelevant-flag-data)), if making everything reconcile is easier that way.

Reply:
@ddbeck Thanks very much. I don't find anywhere near as easy to understand in this context, but I guess it is easier when someone sees the rendered version - as they can look at the first entry and work out the first versions where support exists. 

I have copied this in. No need to drop the older statements. The properties themselves are much easier to document because they have a smaller context to work in. See `clip-path` in top page above. 

I'll add a few comments against the new version :-)